### PR TITLE
Mystical Crop compatibility & missing Butter/Midas gold from Mythic Metals

### PR DESCRIPTION
**Pull request by CantWeAllDisagree**
Renamed Branch. 

Still learning github kekw.

*Edit by Steveplays28: Original PR (#586):*
> Adds compatibility for the [Mystical Nature](https://www.curseforge.com/minecraft/mc-mods/mystical-nature) mod and I also noticed butter was missing from Mythic Metals so i threw that in there too.

### CODE REVIEW

File: content/forgero-compat/gradle.properties
Comment (SigmundGranaas) [inline content/forgero-compat/gradle.properties:1]: All property files can be deleted I believe, they are not used anymore.

File: fabric/gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 minecraft_version=1.19.2
 yarn_mappings=1.19.2+build.28
-loader_version=0.14.12
+loader_version=0.14.19
```
Comment (SigmundGranaas) [inline fabric/gradle.properties]: Nice :+1:

File: content/forgero-vanilla/gradle.properties
Comment (SigmundGranaas) [inline content/forgero-vanilla/gradle.properties:1]: All contents in this file can be deleted

File: content/forgero-structures/gradle.properties
Comment (SigmundGranaas) [inline content/forgero-structures/gradle.properties:1]: Can be deleted

File: content/forgero-extended/gradle.properties
Comment (SigmundGranaas) [inline content/forgero-extended/gradle.properties:1]: Can be deleted

**Review by SigmundGranaas — CHANGES_REQUESTED** [review]
Some leftover gradle.properties files I did not know where present since the grand restructuring. It would be nice to either delete the content or the entire file to not get confused as to if these are actually used. Other than that, great work!

**Comment by SigmundGranaas** [comment]
Also, merge the latest commit to get the gametests fixed.

**Comment by CantWeAllDisagree** [comment]
POG on the gametests

**Review by SigmundGranaas — APPROVED** [review]

File: fabric/gradle.properties (hunk 1 above)
Comment (SigmundGranaas) [inline fabric/gradle.properties]: I take it back, this wasn't nice. I tested it locally and this seems to break the compat module, not necessarily just arrp, as this is loaded in the core module as well.

**Review by SigmundGranaas — CHANGES_REQUESTED** [review]

File: fabric/gradle.properties (hunk 1 above)
Comment (CantWeAllDisagree) [reply fabric/gradle.properties]: reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

**Review by SigmundGranaas — APPROVED** [review]

**Pull request merged**